packages/components/src/components/FieldError/FieldError.tsx

```diff
@@ -14,7 +15,12 @@ export const FieldError: FC<FieldErrorProps> = (props) => {
 
   return (
     <Aria.FieldError {...rest} className={rootClassName}>
-      {children}
+      {({ validationErrors }) => (
+        <>
+          <IconDanger size="s" />
+          {children ? children : validationErrors.join(" ")}
```

Comment:
Würde die Children evtl. noch in einm Element (`span`?) zusammenfassen, damit der Abstand wirklich nur zwischen Icon und Content ist.

```tsx
      <FieldError>
        Invalid input <strong>foo</strong>
      </FieldError>
```